demo-app/src/components/molecules/Loginform/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useState } from "react";
+import TypographyComp from "../../atoms/Typograph";
+import TextFieldComp from "../../atoms/InputField";
+import ButtonOrganism from "../../organisms/ButtonOrganism";
+import { Stack } from "@mui/material";
+import "./index.css";
+const LoginForm: React.FC = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  return (
+    <div id="loginform">
+      <Stack mb={5} spacing={3}>
```

Comment:
don't use inline styling instead you can use styled component..

Reply:
updated